Shaders/deferred_light/deferred_light.frag.glsl

```diff
@@ -224,39 +230,35 @@ void main() {
 	vec2 envBRDF = textureLod(senvmapBrdf, vec2(roughness, 1.0 - dotNV), 0.0).xy;
 #endif
 
-	// Envmap
+// Envmap
 #ifdef _Irr
-
 	vec3 envl = shIrradiance(n, shirr);
-
 	#ifdef _gbuffer2
 		if (g2.b < 0.5) {
 			envl = envl;
 		} else {
 			envl = vec3(1.0);
 		}
 	#endif
-
 	#ifdef _EnvTex
 		envl /= PI;
 	#endif
 #else
 	vec3 envl = vec3(1.0);
 #endif
 
-#ifdef _Rad
-	vec3 reflectionWorld = reflect(-v, n);
-	float lod = getMipFromRoughness(roughness, envmapNumMipmaps);
-	vec3 prefilteredColor = textureLod(senvmapRadiance, envMapEquirect(reflectionWorld), lod).rgb;
-#endif
-
 #ifdef _EnvLDR
 	envl.rgb = pow(envl.rgb, vec3(2.2));
 	#ifdef _Rad
 		prefilteredColor = pow(prefilteredColor, vec3(2.2));
 	#endif
 #endif
 
+#ifdef _Rad
+	vec3 reflectionWorld = reflect(-v, n);
+	float lod = getMipFromRoughness(roughness, envmapNumMipmaps);
+	vec3 prefilteredColor = textureLod(senvmapRadiance, envMapEquirect(reflectionWorld), lod).rgb;
+#endif
```

Comment:
Hm why still move the block? Just from a glance there is `vec3 prefilteredColor`, but `prefilteredColor` is already used above?